packages/api/bin/serve.js

```diff
@@ -428,21 +426,15 @@ async function eraseDynamoTables(stackName, systemBucket) {
   process.env.stackName = stackName;
 
   // Remove all data from tables
-  const providerModel = new models.Provider();
-  const collectionModel = new models.Collection();
   const rulesModel = new models.Rule();
   const executionModel = new models.Execution();
   const granulesModel = new models.Granule();
```

Comment:
Looks like these need removed

Reply:
It might be worth testing serving the API locally. I suspect some of this is out-of-date.